systemtests/tests/py2plug-fd-mariabackup/testrunner

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+set -e
+set -o pipefail
+set -u
+#
+# This systemtest tests the mariadb plugin functionality
+# of the Bareos FD by using the supplied module
+#   BareosFdPluginMariabackup.py
+#
+TestName="$(basename "$(pwd)")"
+export TestName
+
+
+
+JobName=backup-bareos-fd
+#shellcheck source=../environment.in
+. ./environment
+MYQL_CLIENT="${MARIADB_CLIENT_BINARY} --defaults-file=mysqldefaults --user=$USER"
+MARIABACKUP="${MARIABACKUP_BINARY} --defaults-file=mysqldefaults"
+
+JobName=backup-bareos-fd
+#shellcheck source=../scripts/functions
+. "${rscripts}"/functions
+"${rscripts}"/cleanup
+"${rscripts}"/setup
+
+shutdown_mysql_server(){
+[ -f "mysql/mysqld.pid" ] && kill "$(cat mysql/mysqld.pid)" || :
+[ -f "mysql/data/${HOSTNAME}.pid" ] && kill "$(cat mysql/data/${HOSTNAME}.pid)" || :
+}
+
+startup_mysql_server(){
+${MARIADB_DAEMON_BINARY} --defaults-file=mysqldefaults >mysql/mysql.log 2>&1 &
+
+tries=60
+echo "waiting for mysql server to start"
+while ! $MYQL_CLIENT <<< "select version();"  > /dev/null 2>&1; do
+  [ $((tries-=1)) -eq 0 ] && {
+    echo "Could not start mysql server"
+    cat mysql/mysql.log
+    shutdown_mysql_server
+    exit 1
+  }
+  echo -n "."
+  sleep 1
+done
+
+}
+
+
+
+shutdown_mysql_server
+
+# kill running mysqld instance listening on our port
+sudo fuser "${test_db_port}"/tcp -k || :
```

Comment:
We discuss to remove it : if build and tests still work 100%

Reply:
Yes, that was removed in 0d628428747aa363057fa48951b000a833c13005